### PR TITLE
describe how to generate a sample ansible.cfg

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Show how a user can create a sample ansible.cfg

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/templates/config.rst.j2
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by bcoca** [comment]
https://github.com/ansible/ansible/blob/devel/examples/ansible.cfg

File: docs/templates/config.rst.j2 [hunk 1]
```diff
@@ -40,6 +40,21 @@ Ansible will process the above list and use the first file found, all others are
...
+.. _generating_ansible.cfg:
+
+Generating a sample ``ansible.cfg`` file
+-----------------------------------------
+
+You can generate a fully commented-out example ``ansible.cfg`` file, for example::
+
+   $ ansible-config init --disabled > ansible.cfg
+
+You can also have a more complete file that includes existing plugins::
```
Comment (ariordan-redhat) [inline docs/templates/config.rst.j2:52]:
```suggestion
You can also have a more complete file that includes existing plugins:

.. code-block:: shell

```

File: docs/templates/config.rst.j2 [hunk 2]
```diff
@@ -40,6 +40,21 @@ Ansible will process the above list and use the first file found, all others are
         inventory = /etc/ansible/hosts  ; This points to the file that lists your hosts
 
 
+.. _generating_ansible.cfg:
+
+Generating a sample ``ansible.cfg`` file
+-----------------------------------------
+
+You can generate a fully commented-out example ``ansible.cfg`` file, for example::
```
Comment (ariordan-redhat) [inline docs/templates/config.rst.j2:48]:
```suggestion
You can generate a fully commented-out example ``ansible.cfg`` file, for example:

.. code-block:: shell
```

**Review by ariordan-redhat — COMMENTED** [review]
Should the `ansible-config init` lines be wrapped in a `code-block` element to prevent translating the code ?

File: docs/templates/config.rst.j2 (hunk 2 above)
Comment (samccann) [reply docs/templates/config.rst.j2:48]: I tried it that way and it didn't generate the code block, so I stuck with the `::` approach.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**